Oqtane.Server/Pages/_Host.cshtml.cs

```diff
@@ -24,6 +41,24 @@ public void OnGet()
                 ProcessModuleControls(assembly);
                 ProcessThemeControls(assembly);
             }
+
+            // if framework is installed 
+            if (!string.IsNullOrEmpty(_configuration.GetConnectionString("DefaultConnection")))
+            {
+                Uri uri = new Uri(Request.GetDisplayUrl());
+                var alias = _aliases.GetAlias(uri.Authority + "/" + uri.LocalPath.Substring(1));
+                _state.Alias = alias;
+
+                // set default language for site
```

Comment:
If you looked to the commits history this change was here, but I remove it later one, when we support both client and server Blazor, I don't think we need it anymore. Could you please let me know what's the reason that enforce you to add it?

Reply:
Seems you want to set the default language, that stored in the database. I need to try your PR in my laptop before we merge

Reply:
The requirement described previously was for a Site to be able to specify the Languages that it supports and specify which of those Languages is the Default. Then when a user browses to a Site, the framework should set the culture to the default language for the site.

Reply:
I knew but this will break the old localization if I'm not wrong, because server and client blazor using a special cookie name

Reply:
I do not understand what you mean by "old localization". The code for setting the cookie came from the official docs https://docs.microsoft.com/en-us/aspnet/core/blazor/globalization-localization?view=aspnetcore-5.0. I assume that both client and server Blazor use the same cookie name?

Reply:
I mean the current localization, that could you show may brings some issue, tonight I will test and revise your PR ..

Reply:
I am going to merge this PR as it contains some fixes which are required for Languages to install properly. We can always back out the changes to _Host if they are problematic - although I think it is the right approach for setting a default culture per site.

Reply:
@sbwalker as I told you this breaks the localization, try to change a language via the switcher, it will not take an affect coz you enforce  the "Host" to set the default language all the time

I'm trying to fix the issue, that why I told you to hold before you merge